workshops/interactive_maps/README.md

```diff
@@ -0,0 +1,230 @@
+
+![hooray](https://media.giphy.com/media/11sBLVxNs7v6WA/giphy.gif)
+
+##### The Javascript Part
+Finally! The functionality to our interactive map. These is where we use functions. Functions are reusable blocks of code that are callable from our HTML document. There are different types of functions in javascript, here we'll be using the most basic one. A function is written like this,
+```js
+function myFunctionName(parameter){
+      //some code here
+}
+```
+Each function starts with the function keyword. This is followed by the function's name. After this come's the parameters. Parameters are variable values that we can use inside the function. These are specified when the function is being called. Parameters are optional. For example, here is a function that adds two numbers. It has two parameters.
+```js
+function addNumbers(firstNum, secondNum){
+    console.log(firstNum + secondNum)
+}
+```
+And here is a function without a parameter. It simply prints out 'hello world'.
+```js
+function sayHello(){
+    console.log('Hello World!')
+}
+```
+Now that you have a basic understanding of how functions work, head over into your script.js file and add this in.
+```js
+var map = document.getElementById('map');
+function changeImage(location){
+    if (location == 'ocean'){
+        map.src = 'https://cloud-9p8jb8vst.vercel.app/orgocean.png';
+    }
+    else if (location == 'residential'){
+        map.src = 'https://cloud-gga696gyn.vercel.app/orgresidential.png';
+    }
+    
+    else if (location == 'roadway'){
+        map.src = 'https://cloud-bzeicw4mm.vercel.app/orgroad.png';
+    }
+    
+    else if (location == 'vendors'){
+        map.src = "https://cloud-9ldvvvhb4.vercel.app/orgvendors.png";
+    }   
+    
+    else if (location == 'beach'){
+        map.src = 'https://cloud-3bvmpy2n1.vercel.app/orgbeach.png';
+    }
+    else if (location == 'normal'){
+        map.src = 'https://cloud-coh6o3mhy.vercel.app/orgbg.png';
+    }
+}
+```
+This snippet declares a variable named map, which corresponds to the map element in our html file. Then it creates a function named changeImage() with the parameter 'location'. When we call this function from our html file with specific values like 'ocean' or 'residential', (These values passed into parameters are called arguments.) they call on the actual function from the JS file. Inside the function, is an IF -ELSE IF statement. It does exactly what it means. If the location argument is 'beach', set the map's src attribute to the beach highlighted image, and so on. The syntax is
+```js
+if (condition) {
+    Do something;
+}
+else if (another condition) {
+    Do something else;
+}
+else {
+    Do something totally different;
+}
+```
+### Conclusion
+And just like that, you have gone through the basics of HTML/CSS and a bit of JS to create your own interactive map. Now it's up to you to add in a bit of your sparkle and tweak it.
+##### Hacking it
```

Comment:
need three examples with demo links + source code

Reply:
still don't see anything here

Reply:
Oh you mean three example website? Oops, I thought u meant the If statements. Workin on it

Reply:
Heyyo @sampoder, I've pushed another commit. Is this what you wanted?

Reply:
yep, right style just the original demo doesn't count,

thanks